code/modules/chemistry/Chemistry-Machinery.dm

```diff
@@ -420,15 +420,15 @@ TYPEINFO(/obj/machinery/chem_shaker)
 			src.audible_message("<span class='alert'>[src] is rotating a bit too fast!</span>")
 		else
 			src.audible_message("<span class='notice'>[src] whirs to life, rotating its platform!</span>")
-		if (!(src in processing_items))
-			processing_items.Add(src)
+		if (!(src in processing_machines))
+			processing_machines.Add(src)
```

Comment:
not quite that simple, sorry I should have specified that you need to use `SubscribeToProcess` and `UnsubscribeProcess` as `processing_machines` is a multidimensional list full of buckets

Reply:
waa